src/drafts/MarkdownEditor/MarkdownEditor.stories.tsx

```diff
@@ -288,6 +293,113 @@ export const CustomButtons = ({
           <MarkdownEditor.ToolbarButton icon={DiffIcon} onClick={onDiffClick} aria-label="Custom Button" />
           <MarkdownEditor.DefaultToolbarButtons />
         </MarkdownEditor.Toolbar>
+      </MarkdownEditor>
+      <p>Note: for demo purposes, files starting with &quot;A&quot; will be rejected.</p>
+    </>
+  )
+}
+
+export const CustomFooter = ({
```

Comment:
Not required, but initially I had to dig to understand the differences between `CustomFooter` and `CustomFooterActions` given that they both use the action buttons. What do you think about consolidating, or alternatively removing the actions here?

Reply:
I left `CustomFooterActions` story to support the existing implementations with the `MarkdownEditor` that use the custom actions, but I would like to consolidate it to use the `CustomFooter` once its published.